src/type-coercions.md

```diff
@@ -1,10 +1,14 @@
 # Type coercions
 
-Coercions are defined in [RFC 401]. [RFC 1558] then expanded on that.
-A coercion is implicit and has no syntax.
+**Type coercions** are implicit operations that change the type of a value.
+They happen automatically at specific locations and are highly restricted in 
+what types actually coerce.
+
+The [type cast operator], `as`, is not a coersion site. However, most type
+conversions allowed by coersion can also be explicitly performed by `as`.
```

Comment:
The phrasing here seems a little strange to me.  Nothing suggests or implies that `as` is a coercion site. The equivalence doesn't make sense because `as` is an expression, not a location. It also says "most type conversions...", but then doesn't say which are allowed. 

I think it would be best just to keep this simple and to the point.  I still suggest my original wording, something like `For explicit type coercions, see [type cast expressions].`.

Also, the word is spelled "coercion" with a `c`.

Reply:
I guess I'm trying to make the distinction that, if it's explicit, it's not a coercion. You're right that that wording is funky, though.